packages/functional-tests/tests/react-conversion/signup.spec.ts

```diff
@@ -138,7 +138,9 @@ test.describe('severity-1 #smoke', () => {
       await relier.signOut();
     });
 
-    test('signup oauth webchannel (sync mobile)', async ({ target }) => {
+    test('signup oauth webchannel - sync mobile or FF desktop 123+', async ({
```

Comment:
Is FF desktop the planned release for oauth webchannel?

Reply:
It is! Though I think it might end up being 124+ instead of 123+, I left this here since I already referenced 123 in a couple other places and we can change them once we know the number.